packages/pwa-devdocs/src/venia-pwa-concept/design/global-style/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Global style guide
+---
+
+{: .bs-callout .bs-callout-info}
+**Note:**
+This design specification is still in development.
+If have any feedback or would like to join the PWA conversation, please join our [Slack][] channel.
+
+This topic provides the global style rules and values used in the Venia theme.
+Use this guide to style components that will fit within the Venia theme.
+
+## Colors
+
+{: .bs-callout .bs-callout-tip}
+***Tip:** Color values are expressed in RGB format because it translates easier to `rgba()` values than HEX format.*
+
+The Venia theme uses the following color classes in its design:
+
+| CSS class         | RGB value                     | Usage            |
+| ----------------- | ----------------------------- | ---------------- |
+| `black`           | `rgb(0, 0, 0)`                |                  |
+| `venia-border`    | `rgb(224, 224, 224)`          | Border color     |
+| `venia-grey`      | `rgb(246, 246, 246)`          |                  |
+| `venia-teal`      | `rgb(0, 134, 139)`            | Buttons          |
+| `venia-text`      | `rgb(33, 33, 33)`             | General text     |
+| `venia-text-alt`  | `rgb(117, 117, 117)`          | Alternative text |
+| `venia-text-hint` | `rgb(158, 158, 158)`          | Hint text        |
+| `mask.root`       | `rgb(0, 0, 0)` at 50% opacity | Mask overlay     |
+{:style="table-layout:auto"}
+
+## Font
+
+Venia uses the open source [Muli font][] from Google in its design.
+
+{: .bs-callout .bs-callout-info}
+***Note:** If you are developing Venia themed components, you must install this font locally because the Muli font is currently not served by the backend server.*
+
+If the browser cannot find or use this font, it defaults to the following OS specific fonts:
+
+* **San Francisco** for Apple users
+* **Roboto** for Android users
+* **Segoe** for Windows users
+
+### Size specifications
+
+The Venia theme uses a root font size of **16px**.
+All other elements, except images and media queries, use this base value to calculate their relative sizes using `rem`.
+
+Examples using **16px** as the root font size:
+
+| `rem` size values        | Calculated size        |
+| ------------------------ | ---------------------- |
+| `padding: 1rem;`         | `padding: 16px;`       |
+| `border-radius: 1.5rem;` | `border-radius: 24px;` |
+| `font-size: 0.75rem;`    | `font-size: 12px;`     |
+{:style="table-layout:auto"}
+
+### Font weights
+
+The Venia theme uses a limited set of font weights in its design.
+For optimal component performance, use no more than 3 font weight variations.
+
+![Muli font weights][]{:width="350px"}
```

Comment:
This shows all of the font weights. Is there a way to only show the three we use?

Reply:
Not sure it's practical to have 600+700, but no italic style. The budget should change or I'd drop the 600 to suit the common bold/700 need.

Typeface weights aren't that unique, so this could be a pain to change across projects if Venia chooses a non-700 weight for the "bold" visual style.

Reply:
Thanks for the comment @brendanfalkowski we will revisit the specs.

Reply:
I think `600` is generally the proper weight for semibold UI text, regardless of the font. It's at least true for every font in our font family. `700` is too strong for UI text.

Initially, I had suggested we only use two styles: normal `400` and semibold `600`. `700` is only included here because it may be useful for overlaying hero images. The Venia designs don't call for italics, and I don't anticipate that they will.

That said, CMS content will probably expect additional styles to be available, including italics. Perhaps we can load additional styles asynchronously. That's how a PWA should do it anyway. 😅